magic.js

```diff
@@ -1227,34 +1225,36 @@ function uid(sec, cb) {
  * same and false if not.
  * @function
  * @api public
- * 
+ *
  * @param {string} input - The first string to check
  * @param {string} ref - The reference string to check against
  * @returns {Boolean} - true if they match, false otherwise
  */
 module.exports.util.timingSafeCompare = (input, ref) => {
+  let inputString, refIsString;
```

Comment:
@gkwang shouldn't this be `inputIsString` instead of `inputString` ?

Reply:
oops, yes you're right.